VoxelSniperCore/src/main/java/com/github/kevindagame/snipe/SnipeTool.java

```diff
@@ -29,7 +29,7 @@ protected SnipeTool(Class<? extends IBrush> currentBrush, SnipeData snipeData) {
         snipeData.setVoxelMessage(messageHelper);
 
         IBrush newBrushInstance = instanciateBrush(currentBrush);
-        if (snipeData.owner().getPlayer().hasPermission(newBrushInstance.getPermissionNode())) {
+        if (snipeData.owner().getPlayer().hasPermission(newBrushInstance.getPermissionNode()) || snipeData.owner().getPlayer().hasPermission("voxelsniper.brush.*")) {
```

Comment:
IS this necessary? doesn't voxelsniper.brush.* Contain all brush permissions? (also applies to the other changes in this file)

Reply:
Yes because currently voxelsniper.brush.* doesn't work on master